src/mono/mono/mini/mono-private-unstable.h

```diff
@@ -47,6 +67,10 @@ monovm_initialize (int propertyCount, const char **propertyKeys, const char **pr
 MONO_API int 
 monovm_runtimeconfig_initialize (MonovmRuntimeConfigArguments *arg, MonovmRuntimeConfigArgumentsCleanup cleanup_fn, void *user_data);
 
+// The wrapper MonoCoreRuntimeProperties struct can be stack-allocated or freed, but any of the structs inside it will *not* be copied and need to be heap-allocated
```

Comment:
nit: maybe be more explicit that they have to be heap-allocated and can't be freed at all.

Reply:
Sure